### DISCUSSION TITLE
OpenAudible only finds 139 of the 147 titles

### DISCUSSION THREAD

**Discussion by cypres0099** [openaudible/openaudible#1076] [Q&A] [answered]
Ran my first full sync, download, convert on my library and it found and downloaded 139 titles, but 8 are missing. One that's missing is "48 Laws of Power", which I think it just a regular audiobook. 

Any ideas on why the quick and full scans are only finding 139 titles?

**Comment by oboe45** [comment]
This happens most often for "included" titles (free ones), or non-books
(magazine subscriptions etc) or - more often than seems reasonable. the
title was republished and shows up as "not available".  If you cannot
download or stream any of the missing books, contact Audible - there were
at least 40 books in my 1000 plus catalog that were "not available" but for
which I found the purchase item in my account.  Audible credited for the
missing titles.

**Comment by openaudible** [comment, marked as answer]
You need to go back and make sure that "48 Laws of Power" (or whatever book is missing) is still available and accessible in your library. (You can play it from the audible app and it shows in the audible web browser.) You can also try a "Full Rescan" and look at the Console to see if the book is mentioned for getting skipped for any reason.
